llvm/test/CodeGen/AMDGPU/phi-moveimm-subreg-input.mir

```diff
@@ -0,0 +1,30 @@
+# RUN: not --crash llc  -mtriple=amdgcn -mcpu=gfx900 -verify-machineinstrs -run-pass=si-fix-sgpr-copies -o - %s 2>&1 | FileCheck -check-prefix=GCN %s
+
```

Comment:
This looks like an unrelated test?

Reply:
In this specific testcase(phi-vgpr-input-moveimm.mir) it is throwing illegal copy error as expected, and it is because isSSA considers subreg defs invalid in SSA. So I have extracted the specific subtest & converted it into not crash. Please let me know if this is valid.

Reference : https://github.com/llvm/llvm-project/blob/main/llvm/lib/CodeGen/MIRParser/MIRParser.cpp#L361

Reply:
@arsenm Please take a look at this testcase change.

Reply:
I guess this test should be removed. See my comment below in the original test.

Reply:
I removed this testcase instance.